data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+
+```
+<a href="https://piwik.pro/document-url" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/document-url', 'download']);">Download document</a>
+```
+
+Tracking downloads when using log importer
+""""""""""""""""""""""""""""""""""""""""""
+
+When you use the :ref:`Log Importer<data-collection-web-log-analytics>`, files with one of the file extensions listed above will be automatically tracked as downloads in Piwik PRO.
+
+Outlinks
+^^^^^^^^
+The Piwik PRO `Outlinks` report shows the list of external URLs that were clicked by your visitors. Outlinks are links that have different domain than those configured for your website. For example, if your visitor click on a link to `piwik.pro` and your website domain is `example.org`, this will be reported as an outlink, no matter if the website opens in current tab/window or a new one. For each outlink, Piwik PRO will report the number of clicks, and the number of unique clicks. Unique clicks are the number of unique visitors that clicked to a given domain. If one visitor clicks multiple times on one outlink it will be reported as one unique click and multiple clicks. For example `http://piwik.pro/` will have 5 clicks and 1 unique click. 
+
+```<a href="https://piwik.pro">Piwik PRO</a>```
+
+Confguring which domains are outlinks
+"""""""""""""""""""""""""""""""""""""
+
+If you want to track links on your subdomains as a regular PageViews, for example whan your main page is `piwik.pro` and you want to track views of `blog.piwik.pro` as a regular PageViews not outlinks, you have to confgure tracker to recognize this additional domain. You can do it in two ways.
+
+You can configure it in website settings section of the Administration panel. Go to the Administration > Websites & apps > Settings > General settings > URLs. Add all the domains that should not be treated as outlinks.
+
+[Here is the screenshoot of Administration panel]
+
+You can use :ref:`setDomains()<ignoring-alias-domains>` function of JS tracker.
+
+```
+_paq(['setDomains', ["blog.piwik.pro", "piwik.pro", "*.other-domain.pro"]]);
+
+_paq.push(['trackPageView']);
+```
+
+.. note::
+  This will overwrite URLs configured in Administration panel, so use it wisely.
+
+Marking links as outlinks in HTML code
+""""""""""""""""""""""""""""""""""""""
+
+Similar as downloads, links can be set to be treated as outlinks manually, but only with CSS classes, you cannot use a HTML attribute.
+
+You can use one of default CSS classes: `piwik_link` or `piwik-link`. eg. 
+
+```
+<a href='https://piwik.pro' class="piwik-link">Piwik PRO</a>
+```
+
+Or you can define your custom CSS classes for outlinks with ours :ref:`Javascript Tracker API<force-tracking-using-css-class>`. 
+
+```
+// now all clicks on links with the css class "external" will be counted as outlinks
+
+// you can also pass an array of strings
+_paq.push(['setLinkClasses', "custom-link-class"]);
```

Comment:
Should information about all variances be available here or in the description of `setLinkClasses` function (level 4)? I think here we should limit to one example

Reply:
It is guide, so I think yes. I see no sense in guides that show you that something is possible and show only a part of it.